src/bundle/Resources/public/scss/_labels.scss

```diff
@@ -0,0 +1,21 @@
+.ez-label {
+    margin-bottom: 0;
+    font-weight: 700;
+
+    &:after {
+        content: ': ';
+    }
+}
+
+.ez-label.checkbox-inline,
+.ez-label.form-check-label {
+    margin-bottom: .0625rem;
+
+    &:after {
+        content: '';
+    }
+}
+
+.ez-label.form-check-label:not(.checkbox-inline) {
```

Comment:
If you follow my suggestions above, then it should be placed inside the `&.form-check-label {` selector, like this: `&:not(.checkbox-inline)`

Reply:
Perfect. Thanks.